README.md

```diff
@@ -1,223 +1,186 @@
-# Command-line test framework
+## Icetea test framework
 
-*Icetea* is an [mbed](www.mbed.com) test framework written
-with python 2.7. It is generally used to verify the ARM mbed
+*Icetea* is an [Mbed](https://www.mbed.com) test framework written
+with python. It is generally used to verify the ARM mbed
 IoT Device Platform provides the operating system and cloud services.
```

Comment:
@peknis01 is this right?

Reply:
Seems to go to the proper page. If you are asking about the link.

Reply:
I was thinking this "cloud service" sentence.. I am not sure how it should be in Pilion time

Reply:
Pelion Device Management (get rid of mbed, unless you are talking about Mbed OS, TLS or CLI). After the first mention, drop out Pelion and just talk about Device Management, when referring to the former Mbed Cloud.